### PR TITLE
FOGL-7498: show blank value for config items if user changed it to blank

### PR DESCRIPTION
**Pull request by monika-Dianomic**

### CODE REVIEW

File: src/app/services/configuration-control.service.ts [hunk 1]
```diff
@@ -145,7 +145,7 @@ export class ConfigurationControlService {
     Object.keys(configuration).forEach(key => {
       const element = configuration[key];
       element.key = key;
-      element.value = element.value ? element.value : element.default;
+      element.value = (element.value || element.value === '') ? element.value : element.default;
```
Comment (praveen-garg) [inline src/app/services/configuration-control.service.ts:148]: should we have (element.value || element?.value === '')

Comment (praveen-garg) [reply src/app/services/configuration-control.service.ts:148]: or much safer 
element.value !== undefined  ? element.value : element.default;

**Review by gnandan — COMMENTED** [review]
is element.key always string type? if not, is it possible to update element.value only for string types ?

**Comment by monika-Dianomic** [comment]
@gnandan We always get value cast as string.

**Pull request merged**